modules/vba_documents/lib/vba_documents/upload_validator.rb

```diff
@@ -112,32 +112,28 @@ def validate_line_of_business(lob, submission_version)
       end
     end
 
-    DEFAULT_PDF_VALIDATOR_OPTIONS = {
-      check_encryption: false # Owner passwords are allowed, user passwords are not
-    }.freeze
-
     def validate_document(file_path, part_name, pdf_validator_options = {})
-      options = DEFAULT_PDF_VALIDATOR_OPTIONS.merge(pdf_validator_options)
-      options.merge!({ check_page_dimensions: false }) if Flipper.enabled?(:vba_documents_skip_dimension_check)
-
-      result = PDFValidator::Validator.new(file_path, options).validate
+      result = PDFValidator::Validator.new(file_path, pdf_validator_options).validate
 
       unless result.valid_pdf?
         errors = result.errors
 
         if errors.grep(/#{PDFValidator::FILE_SIZE_LIMIT_EXCEEDED_MSG}/).any?
-          raise VBADocuments::UploadError.new(code: 'DOC106',
-                                              detail: 'Maximum document size exceeded. Limit is 100MB per document')
+          code = 'DOC106'
+          detail = CentralMail::UploadError.default_message(code, pdf_validator_options)
```

Comment:
I feel silly for this, but as far as I can see, this was the center of the missing error details - I guess I hadn't noticed that we were explicitly setting the `detail` here (vs. the `message`) before. Even though the UploadError was setting a message, my previous updates neglected to keep the `detail` intact. As far as I can tell, the behavior now matches the old behavior - allowing `CentralMail::UploadError` to set the `message`, but also setting the `detail` here.

Reply:
This is just a suggestion, and I'm not sure if I feel it's actually better, but structuring the code the following way would remove the need for `default_message` and for the second call to the `CentralMail::UploadError` class.

```
upload_error = VBADocuments::UploadError.new(code: 'DOC106', detail: 'Maximum document size exceeded. Limit is 100MB per document')
code = upload_error.code
detail = upload_error.detail

raise upload_error
```